### ISSUE TITLE
Add option to show main process output in the renderer

### ISSUE THREAD

**Issue by sindresorhus** [sindresorhus/electron-timber#15]
Can you be useful for debugging a production app, since you wouldn't otherwise easily see the logging from the main process.

@acheronfail Thoughts?

**Comment by cgalvarez** [comment]
In order to see the main process messages you can execute the built app from the command line like `path/to/app/folder $ ./app_executable`

**Comment by sindresorhus** [comment]
@cgalvarez Yup, but that's inconvenient. Also nice to have all the logging in one place when debugging.

**Comment by sindresorhus** [comment]
See https://github.com/sindresorhus/electron-timber/pull/19 for inspiration if you want to work on this issue.
